spynnaker/pyNN/models/neuron/synapse_dynamics/synapse_dynamics_structural_common.py

```diff
@@ -615,12 +632,16 @@ def __write_post_to_pre_table(self, spec, app_vertex, post_slice,
                     pre_vertex_slice = graph_mapper._slice_by_machine_vertex[
                         row[2].pre_vertex]
                     pre_vertex_id = source - pre_vertex_slice.lo_atom
-                    masked_pre_vertex_id = pre_vertex_id & (2 ** 17 - 1)
+                    masked_pre_vertex_id = \
+                        pre_vertex_id & \
+                        SynapseDynamicsStructuralCommon.BIT_MASK_16_BIT
```

Comment:
This isn't the same as it was previously (2 ** 17 - 1 vs 2 ** 16 - 1).  Is this correct?

Reply:
This is correct now (previously a bug). For clarity, the definition of BIT_MASK_X_BIT could be replaced with 0xFF (for 8bit) and 0xFFFF (for 16bit).